benchmarks/stepper/analyze.jl

```diff
@@ -217,14 +352,16 @@ function make_subtime_tables(all_df)
 end
 
 function make_perf_profiles(all_df, comp, metric)
-    pp = filter(t -> t.solver_options in comp, all_df)
+    pp = filter(t -> t.enhancement in comp, all_df)
+
     # BenchmarkProfiles.jl expects NaNs for failures
     select!(pp,
         :inst_key,
-        :solver_options,
+        :enhancement,
         [metric, :conv] => ByRow((x, y) -> (y ? x : NaN)) => metric,
         )
-    wide_df = unstack(pp, :solver_options, metric)
+    wide_df = unstack(pp, :enhancement, metric, allowduplicates = true)
```

Comment:
why?

Reply:
from memory a duplicate here would indicate some instance had duplicate data, which shouldn't happen, so an error would be good